src/middleware/index.ts

```diff
@@ -182,26 +292,16 @@ const validateStacOrThrow = (query: StacQuery) => {
  * Middleware that validates query params.
  */
 export const validateStacQuery = (
-  req: Request<object, object, object, StacQuery>,
+  req: Request<object, StacQuery, object, StacQuery>,
   _res: Response,
   next: NextFunction
 ) => {
-  const query = req.query;
-  validateStacOrThrow(query);
-
-  next();
-};
-
-/**
- * Middleware that validates a STAC POST body.
- */
-export const validateStacBody = (
-  req: Request<object, StacQuery, object, object>,
-  _res: Response,
-  next: NextFunction
-) => {
-  const query = req.body;
-  validateStacOrThrow(query);
+  // this feels hacky, check express decoding for proper handling of "+" symbol
+  // this is for timestamps with offsets i.e. 1937-01-01T12:00:27.87+01:00
+  if (req.query.datetime) {
+    req.query.datetime = req.query.datetime.replace(" ", "+");
```

Comment:
Datetime using `+` as the timezone offset were being handled incorrectly resulting in an invalid datetime during decoding and failing when being sent to CMR.